sdk/communication/communication-common/src/identifierModels.ts

```diff
@@ -206,6 +259,30 @@ export const getIdentifierRawId = (identifier: CommunicationIdentifier): string
       }
       return `8:orgid:${microsoftTeamsUserId}`;
     }
+    case "microsoftTeamsBot": {
+      const { microsoftTeamsBotId, rawId, cloud, isGlobal } = identifierKind;
+      if (rawId) return rawId;
+      if (isGlobal) {
+        switch (cloud) {
+          case "dod":
+          return `28:dod-global:${microsoftTeamsBotId}`;
+        case "gcch":
+          return `28:gcch-global:${microsoftTeamsBotId}`;
+        case "public":
+          return `28:${microsoftTeamsBotId}`;
+        }
+        return `28:${microsoftTeamsBotId}`;
```

Comment:
Instead of duplicating code on line 272 and 274, we can have a default case
Line 271 : 
```
default :
  return `28:${microsoftTeamsBotId}`;
```

Reply:
Thanks, updated!